draft-ietf-acme-acme.md

```diff
@@ -475,6 +475,7 @@ in the "type" field (within the "urn:ietf:params:acme:error:" namespace):
 | badNonce              | The client sent an unacceptable anti-replay nonce                              |
 | badSignatureAlgorithm | The JWS was signed with an algorithm the server does not support               |
 | invalidContact        | The contact URI for an account was invalid                                     |
+| unsupportedContact    | The contact URI for an account used an unsupported protocol scheme             |
```

Comment:
Good point. I will update the wording for `invalidContact` and `unsupportedContact`.